### PR TITLE
Add new file : getting-closer.md

### PR DESCRIPTION
**Pull request by gaeulautumn**
Modify approached-green-people.md

### CODE REVIEW

**Comment by gaeulautumn** [comment]
Please ! I need your quick review! @casey-collab

File: korean/explore-outside/investigate-marshmallow-structure/approach-green-people/getting-closer.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
```
Comment (casey-collab) [inline korean/explore-outside/investigate-marshmallow-structure/approach-green-people/getting-closer.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by gaeulautumn** [comment]
I modified the directory. Please review this @casey-collab

File: korean/explore-outside/investigate-marshmallow-structure/approach-green-people/approached-green-people.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
 초록 인간들에게 다가가자 그들이 고개를 돌려 당신이 있는쪽을 바라보고 있어요.
 당신은 어떻게 할까요?
 
-[마시멜로 방 밖으로 되돌아간다] (../../explore-outside.md)
\ No newline at end of file
+[마시멜로 방 밖으로 되돌아간다] (../../explore-outside.md)
+
+[초록 인간들에게 더 가까이 가본다] (./getting-closer.md)
```
Comment (casey-collab) [inline korean/explore-outside/investigate-marshmallow-structure/approach-green-people/approached-green-people.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: korean/explore-outside/investigate-marshmallow-structure/approach-green-people/getting-closer/getting-closer.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+
```
Comment (casey-collab) [inline korean/explore-outside/investigate-marshmallow-structure/approach-green-people/getting-closer/getting-closer.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Pull request merged**